card.ml

```diff
@@ -73,3 +78,121 @@ let color c = c.color
 let actions c = c.actions
 
 let amount c = c.amount
+
+let digit c = c.digit
+
+(********************************************************************)
+(* Standard cards *)
+(********************************************************************)
+
```

Comment:
Not good to place data in the Card module. Standard UNO cards should be packaged with the game in the form of a JSON. We already have a method for parsing cards from JSONs, so why not just use those? With the current implementation that is here, you really are doing unnecessary work